src/shared/validation/use-imperative-validate.js

```diff
@@ -12,7 +12,7 @@ import {
  * @params {Object} options
  * @params {Boolean} options.enabled - Defaults to `true`
  **/
-export default function useImperativeValidate() {
+export const useImperativeValidate = () => {
```

Comment:
this is not necessary FYI, but I was having some trouble with jest mocking when it was a default, so I changed to be a named export.